src/index.js

```diff
@@ -83,6 +94,17 @@ class LndEngine {
       this[configKey] = this.currencyConfig[configKey]
     })
 
+    // 90 minutes, from BOLT #2
+    // see https://github.com/lightningnetwork/lightning-rfc/blob/master/
+    //     02-peer-protocol.md#cltv_expiry_delta-selection
+    this.outboundTimeLock = outboundTimeLock || 9 * this.secondsPerBlock
+
+    // based on a forward time lock delta of 40 blocks from LND
+    // see https://github.com/lightningnetwork/lnd/pull/2759
+    this.retrieveWindowDuration =
+      retrieveWindowDuration || 10 * this.secondsPerBlock
```

Comment:
same comment - may make sense to define this in seconds rather than blocks given that this engine supports two blockchains

Reply:
I don't know if there is a formula for this one, but it kind of makes sense that blockchain congestion would be relieved in proportion to block frequency.

Reply:
perhaps for publish duration, but for retrieval I'd argue that it's more likely about network connectivity or downtime.

Reply:
Ok I think that makes sense, I updated the retrieveWindowDuration to be independent of block time